README.md

```diff
@@ -0,0 +1,68 @@
+cerner_splunk_ingredient Cookbook
+=================================
+Resource cookbook which provides custom resources for installing Splunk.
+
+These resources can:
+- Install Splunk via downloadable package or archive
+
+Requirements
+------------
+Supports Linux and Windows based systems with package support for Debian, Redhat,
+and Windows.
+
+---
+
+Resources
+---------
+
+### splunk_install
+Installs Splunk. Easy.
```

Comment:
"Use the splunk_install resource to manage the core parts of a Splunk installation" perhaps? The resource itself doesn't install splunk, using one of the actions you take on the resource asserts that the desired version of Splunk is installed (subject to package manager constraints of course). (Also I'm not sure if I'm crazy or not with this idea, but I wonder if like with ReST, Resources should be Nouns, and you perform Action (verbs) upon them.)